zhaquirks/osram/switchmini.py

```diff
@@ -0,0 +1,129 @@
+"""Osram Smart+ Switch Mini device."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    LevelControl,
+    OnOff,
+    Ota,
+    PowerConfiguration,
+    Scenes,
+    PollControl,
+)
+from zigpy.zcl.clusters.lighting import Color
+from zigpy.zcl.clusters.lightlink import LightLink
+
+from . import OSRAM
+from ..const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    SHORT_PRESS,
+    COMMAND,
+    COMMAND_ON,
+    MODELS_INFO,
+    BUTTON_1,
+    ENDPOINT_ID,
+    COMMAND_STEP_ON_OFF,
+    COMMAND_STOP,
+    BUTTON_2,
+    BUTTON_3,
+    LONG_RELEASE,
+    LONG_PRESS,
+    COMMAND_MOVE_TO_LEVEL_ON_OFF,
+    COMMAND_OFF,
+    COMMAND_MOVE,
+)
+
+OSRAM_DEVICE = 0x0810
+OSRAM_CLUSTER = 0xFD00
+
+
+class OsramSwitchMini(CustomDevice):
+    """Osram Smart+ Switch Mini device."""
+
+    signature = {
+        MODELS_INFO: [(OSRAM, "Lightify Switch Mini")],
+        ENDPOINTS: {
+            # <SimpleDescriptor endpoint=1 profile=260 device_type=2064
+            # device_version=1
+            # input_clusters=[0, 1, 20, 4096, 64758]
+            # output_clusters=[3, 4, 5, 6, 8, 25, 768, 4096]>
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: OSRAM_DEVICE,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    PowerConfiguration.cluster_id,
+                    PollControl.cluster_id,
+                    LightLink.cluster_id,
+                    OSRAM_CLUSTER,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    LevelControl.cluster_id,
+                    Ota.cluster_id,
+                    Color.cluster_id,
+                    LightLink.cluster_id,
+                ],
+            },
+            # <SimpleDescriptor endpoint=2 profile=260 device_type=2064
+            # device_version=1
+            # input_clusters=[0, 4096, 64768]
+            # output_clusters=[3, 4, 5, 6, 8, 768, 4096]>
+            2: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: OSRAM_DEVICE,
```

Comment:
Import https://github.com/zigpy/zigpy/blob/29e362563f31bc8e77462ac5b2fbd78c40aa926f/zigpy/profiles/zha.py#L64 and use that